src/tasks/start/start-frontend.ts

```diff
@@ -73,13 +80,16 @@ Client:  ${clientURL}`)
       )
     })
 
-  onExit(() => {
-    srcWatcher.close()
-    artifactWatcher.close()
-  })
-
-  emitEvent(FRONTEND_STARTED_SERVING, 1000)
-
   logFront('Watching changes on front end...')
-  await startAppWatcher(appSrcPath)
+  const { close: closeWatchScript } = await startAppWatcher(appSrcPath)
+
+  return {
+    close: (): void => {
```

Comment:
I agree that it would have been the best option, but I found no other way to fix the tests than refactoring them to run in the main process and closing open handlers to continue gracefully